hack/chains/check-taskrun.sh

```diff
@@ -114,29 +120,17 @@ say $SIGNATURE
 pause
 
 title Taskrun payload
-[[ -z $QUIET ]] && echo "$PAYLOAD" | yq-pretty
+[[ -z $QUIET ]] && echo "$RAW_PAYLOAD" | yq-pretty
 
 pause
 
-# Keep going if the verify fails
-set +e
-
-##
-## Fixme: This only works with artifacts.taskrun.format set to 'tekton'.
-## I've never been able to use cosign verify-blob to verify a task's
-## payload and signature using the 'in-toto' format and I don't know why.
-##
 # Now use cosign to verify the signed payload
 title Taskrun verification
 show-then-run cosign verify-blob --key $SIG_KEY --signature $SIG_FILE $PAYLOAD_FILE
-COSIGN_EXIT_CODE=$?
 
 # For debugging...
 title "To view taskrun"
 say " env EDITOR=view kubectl edit $TASKRUN_NAME"
 
 # Clean up
 rm $SIG_FILE $PAYLOAD_FILE
-
-# Use the exit code from cosign
-exit $COSIGN_EXIT_CODE
```

Comment:
We still want this I think, because it might fail for other reasons such as a hacker trying to inject some malware .. :)

Reply:
I don't think this is anymore needed because `set +e` was also removed and the script already has `set -e`. If `cosign verify-blob` fails, the script will end immediately with the error code.

Reply:
What @lcarva said :)